docs/hello_nf-core/02_rewrite_hello.md

```diff
@@ -279,6 +277,21 @@ Learn how to make a simple workflow composable as a prelude to making it nf-core
 
 ## 2. Make the original Hello Nextflow workflow composable
 
+Before we can integrate our workflow into the nf-core scaffold, we need to make it **composable**.
+A composable workflow must be called from a parent workflow—it cannot run on its own—which is exactly how the nf-core template is structured.
+
+### What does the Hello Nextflow workflow do?
+
+If you haven't completed the [Hello Nextflow](../hello_nextflow/index.md) training, here's a quick overview of what this simple workflow does:
+
+1. **Reads greetings** from a CSV file (e.g., "Hello", "Bonjour", "Holà")
+2. **Says hello** by adding "world!" to each greeting
+3. **Converts to uppercase** (e.g., "HELLO WORLD!")
+4. **Collects results** into a single file
+5. **Adds ASCII art** using cowpy to display the final output with a fun character
+
+The workflow uses four Nextflow processes organized into separate module files, takes an input CSV file of greetings, and produces a whimsical output file.
```

Comment:
This is such an odd summary of the workflow. A human would say the workflow takes a CSV file containing greetings, then runs four consecutive steps to transform the greetings and write them out as part of an ASCII picture. Then maybe add that the four steps are done by modularized processes or something to that effect.